novawallet/Common/Protocols/PurchaseHandler.swift

```diff
@@ -0,0 +1,68 @@
+import SoraFoundation
+
+protocol PurchaseHandler {
```

Comment:
I would call this PurchaseFlowManaging. We also have a logic that handles purchase deeplinks and we might confuse in future.